.eslintrc

```diff
@@ -1,176 +1,176 @@
 {
```

Comment:
I know it is a painful thing to check diffs with indentation changes on github =(. I did not touch a single rule (though, I would add some more ecmaFeatures, like in https://github.com/nkbt/esnext-quickstart/blob/master/src/.eslintrc)